py3status/modules/twitch_streaming.py

```diff
@@ -62,7 +65,7 @@ def is_streaming(self):
                 'cached_until': self.py3.CACHE_FOREVER
             }
 
-        r = requests.get('https://api.twitch.tv/kraken/streams/' + self.stream_name)
+        r = requests.get('https://api.twitch.tv/kraken/streams/' + self.stream_name, headers={'Client-ID': self.twitch_client_id})
```

Comment:
@DaeronAlagos It'd be great to get this module working again, if you change this to

```python
        r = requests.get(
            'https://api.twitch.tv/kraken/streams/' + self.stream_name,
            headers={'Client-ID': self.twitch_client_id}
        )
```

it'll fix the flake8 error